shared/insideResinOS/2.0.md

```diff
@@ -0,0 +1,154 @@
+
+## Variants of ResinOS
+
+ResinOS currently comes in 3 different variants all built from the same source but with slightly differing features enabled or disabled. Each version of resinOS produces the following variants:
+
+| Version Name | Variant Type | Description |
+|:------------:|:------------:|:------------:|
+| 2.0.0.rev1   | production   | This is the production version of the resin.io managed OS. This is the OS you should use for any production fleet deployments |
+| 2.0.0.rev1-dev | development | This version the development version of the above and should be used when you are developing a new application and want to use the fast [local mode][local-mode-link] workflow |
+| 2.0.0.rev1-standalone | standalone | This version is an unmanaged version of resinOS, it doesn't not have the resin.io supervisor and does not connect to the management console. Use this variant if you want a stable linux OS to run docker on. |
+
+### Dev vs. Prod images
+
+The Development images are recommended while getting started with resinOS and building a system.
+The dev images enable a number of useful features while developing, namely:
+
+* Passwordless SSH into resinOS on port `22222` as the root user. So one can do `ssh root@<DEVICE_IP> -p22222` and poke around to see how the system runs.
+* Docker socket exposed on via port `2377`, which allows `resin local push` to do remote docker builds on the target device.
+* Getty console attached to tty1 and serial.
+* Capable of entering [local mode][local-mode-link] for rapid development of application containers locally.
+
+__Note:__ Raspberry Pi devices don’t have Getty attached to serial.
+
+The production images have all of the above functionality disabled by default. In both forms of the OS we write logs to an 8 MB journald RAM
+buffer in order to avoid wear on the flash storage used by most of the supported boards. However, persistent logging can be enabled by setting
+the `"persistentLogging": true` key in the `config.json` file in the boot partition of the device.
+
+Both Prod and Dev variants will also allow the setting of a custom hostname via the `config.json`, just add `"hostname": "my-new-hostname"`. Your device will then broadcast (via Avahi) on the network as `my-new-hostname.local`. If you don't set a custom hostname, the device will default to `<short-UUID>.local`.
+
+On the production variant, nothing is written to tty1, on boot up you should only see the resin logo on the HDMI screen and this will persist until your application code takes over the framebuffer. If you would like to replace the resin logo with your own custom splash logo, then you will need to replace splash/resin-logo.png file that you will find in the first partition of our images (boot partition or `resin-boot`) with your own image. NOTE: As it currently stands plymouth expects the image to be named `resin-logo.png`.
```

Comment:
Should be consistent in capitalizing (or not) Development, Production, Standalone, Unmanaged